packages/admin-ui-core/src/createTheme.ts

```diff
@@ -1,6 +1,11 @@
 import { pick, omit, get } from '@vtex/admin-ui-util'
 
 const constants = {
+  /**
+   * When converting tokens with those namespaces to CSS Variables the algorithm will look
+   * to the color values and replace all usage of color arbitrary values with its correspondent color variable.
```

Comment:
I think **by** it is better for this case
```suggestion
   * to the color values and replace all usage of color arbitrary values by its correspondent color variable.
```

Reply:
Replace with is ok too

Reply:
I've changed this a little bit and I think `with` fits better, can you take a look?